pages/js_docs.mdx

```diff
@@ -3,14 +3,54 @@ export default Layout;
 
 # Js
 
-This library provides bindings and necessary support for JS FFI.
-It contains all bindings into Js namespace.
+The [Js](/js_docs) module mostly contains Reason bindings to *standard JavaScript APIs*
```

Comment:
Also note that the JS namespace is supposed to be a zero-abstraction interop layer for cases where users really need to get the JS equivalent for some functions

Reply:
Right, good point!